library/core/src/main/java/com/google/android/exoplayer2/metadata/vorbis/VorbisCommentFrame.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.exoplayer2.metadata.vorbis;
+
+import static com.google.android.exoplayer2.util.Util.castNonNull;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import androidx.annotation.Nullable;
+import com.google.android.exoplayer2.metadata.Metadata;
+
+/** Base class for Vorbis Comment Frames. */
+public class VorbisCommentFrame implements Metadata.Entry {
+
+  /** The frame key and value */
```

Comment:
When generating HTML Javadoc, this is going to attach "The frame key and value" to `key`, and nothing to `value`. Could you document each of them separately?

Reply:
Done